External/FEXCore/Source/Interface/Core/CPUID.cpp

```diff
@@ -644,27 +899,46 @@ FEXCore::CPUID::FunctionResults CPUIDEmu::Function_8000_0001h(uint32_t Leaf) {
   return Res;
 }
 
-constexpr char ProcessorBrand[48] = {
+static constexpr size_t const_strlen(const char *str) {
```

Comment:
The stdlib has an equivalent of this as `std::char_traits<char>::length()`

Reply:
Resolved.